xdrip-get-entries.sh

```diff
@@ -1390,16 +1379,18 @@ function calculate_noise()
     fi
   fi
 
-  if [ $(bc <<< "$variation >= 30") -eq 1 -o  $(bc  <<< "$variation <= -30") -eq 1 ]; then
-      noiseSend=4  
-      noiseString="Heavy"
-      log "setting noise to heavy because - filtered/unfiltered variation of $variation exceeds 30%"
-  fi
-
-  if [ "$orig_status" == "Temporary Session Error" ]; then
+  if [ $(bc <<< "$variation >= 35") -eq 1 -o  $(bc  <<< "$variation <= -35") -eq 1 ]; then
       noiseSend=4  
       noiseString="Heavy"
-      log "setting noise to heavy because tx is in temporary session error"
+      log "setting noise to $noiseString because - filtered/unfiltered variation of $variation exceeds 35%"
+  elif [ $(bc <<< "$variation >= 30") -eq 1 -o  $(bc  <<< "$variation <= -30") -eq 1 ]; then
+      noiseSend=3  
+      noiseString="Medium"
+      log "setting noise to $noiseString because - filtered/unfiltered variation of $variation exceeds 30%"
+  elif [ $(bc <<< "$variation >= 25") -eq 1 -o  $(bc  <<< "$variation <= -25") -eq 1 ]; then
+      noiseSend=3  
```

Comment:
Should this be 1?

Reply:
Yes, good catch

Reply:
actually, it should be 2 for Light noise